### PR TITLE
Fix enable setup

### PR DESCRIPTION
**Pull request by itziakos**
This PR fixes issues similar to #179 and provides a more details on the build and runtime requirements

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,17 +64,36 @@ Kiva currently implements the following features:
...
+
+.. rubric:: Runtime:
+
+- `Numpy <http://pypi.python.org/pypi/numpy>`_
+- `PIL <http://www.pythonware.com/products/pil>`_
+- `traits 4.5.0 <https://pypi.python.org/pypi/traits>`_
+- `traitsui 4.4.0 <https://pypi.python.org/pypi/traitsui>`_
+- `pyface 4.5.0 <https://pypi.python.org/pypi/pyface>`_
+- `apptools 4.3.0 <https://pypi.python.org/pypi/apptools/>`_
+- `kiwisolver <https://pypi.python.org/pypi/kiwisolver>`_
```
Comment (jwiggins) [inline README.rst]: This is optional

**Comment by jwiggins** [comment]
Wait... So the solution is to not leave it up to setuptools?

**Comment by itziakos** [comment]
The problems appears when we are using `setup_requires=['cython'], and we have a latest setuptools installed.  

Please note that `setup_requires` is different from `install_requires` which we still use

**Comment by jwiggins** [comment]
LGTM. Anybody else want to take a look?

**Comment by itziakos** [comment]
@corranwebster is this ok to merge?

**Comment by itziakos** [comment]
ping

**Comment by corranwebster** [comment]
LGTM

**Pull request merged**